Cargo.toml

```diff
@@ -28,27 +28,27 @@ edition = "2021"
 license = "MPL-2.0"
 
 [workspace.dependencies]
+actix-web = "3"
+
 base64 = "0.21"
 cadence = "0.29"
 backtrace = "0.3"
 chrono = "0.4"
 docopt = "1.1"
-env_logger = "0.9"
+env_logger = "0.10"
 futures = { version = "0.3", features = ["compat"] }
 hex = "0.4"
 http = "0.2"
 lazy_static = "1.4"
 protobuf = "=2.25.2" # pin to 2.25.2 to prevent side updating
 rand = "0.8"
 regex = "1.4"
-sentry = { version = "0.19", features = [
-  "with_curl_transport",
-] } # pin to 0.19 until on-prem sentry server is updated
-sentry-backtrace = "0.19"
+sentry = { version = "0.31" }
```

Comment:
I'm guessing we'll still need `with_curl_transport`: IIRC, the [original issue](https://github.com/mozilla-services/syncstorage-rs/issues/289) doesn't trigger an error until you run syncstorage with a `DSN` set and none of our tests go this far.

Reply:
`with_curl_transport` is not a feature of `sentry` 0.31

Reply:
If they just renamed the feature then we'll still need the transport init code in main